compilers/concrete-compiler/compiler/lib/Dialect/FHE/Analysis/ConcreteOptimizer.cpp

```diff
@@ -1002,6 +1003,10 @@ struct FunctionToDag {
     return value.isa<mlir::BlockArgument>();
   }
 
+  bool isMaxNoise(mlir::Operation &op) {
```

Comment:
Should be all tensor operators?

`mlir::isa<mlir::tensor::TensorDialect>(op.getDialect());`